components/pd_client/src/client.rs

```diff
@@ -325,6 +325,8 @@ impl fmt::Debug for RpcClient {
 }
 
 const LEADER_CHANGE_RETRY: usize = 10;
+// periodic request like store_heartbeat, we don't need to retry.
+const NO_RETRY: usize = 1;
```

Comment:
change pd-client v2 as well?

Reply:
make sense, may choose a point to make unify the client, like after partition-raftkv GA.